packages/maskbook/src/extension/options-page/DashboardComponents/CollectibleList/index.tsx

```diff
@@ -155,3 +161,99 @@ export function CollectibleList(props: CollectibleListProps) {
         </CollectibleContext.Provider>
     )
 }
+
+const useCollectibleListAddressStyles = makeStyles((theme) => ({
+    empty: {},
+    button: {},
+}))
```

Comment:
Remove this part if you don't have your own styles. Use only `withClasses` is enough.